### DISCUSSION TITLE
Зацикливание на курьерской доставке

### DISCUSSION THREAD

**Discussion by kaavain** [cdek-it/widget#13] [Ideas]
Я уже писал, но продублирую здесь, так как проблема серьезная.
Если виджет настроен показывать только ПВЗ (в частности у меня - скрывать все кроме ПВЗ и постаматов, не указан город-отправитель, чтобы была только кнопка [ВЫБРАТЬ] без цены), и в "Поиск" по карте ввести адрес до дома, то карта жестко переключается в режим курьерской доставки без какой-либо возможности вернуться к режиму ПВЗ. При этом метки ПВЗ с карты пропадают.
Невозможность вернуться к ПВЗ даже при вводе другого адреса без номера дома, без улицы, особенно критично, так как юзер будет пугаться и убегать.

**Comment by vermorag** [comment]
Исправлено в [3.7.1](https://github.com/cdek-it/widget/releases/tag/v3.7.1)

**Comment by kaavain** [comment]
Замечательно. Только теперь вообще не работает: в обычном режиме (даже после shift-refresh) похоже грузится старая версия, так как при вводе адреса до дома все так же мертвое залипание. А е режиме инкогнито карта не грузится вообще.
<script type='text/javascript' src='https://cdn.jsdelivr.net/gh/cdek-it/widget@latest/dist/cdek-widget.umd.js' charset='utf-8'></script>
Win 11, MS Edge.

**Comment by kaavain** [comment]
Уточню: в MS Edge после нескольких минут ожидания [см.лог] а в Mozilla карта грузится но не работает поиск адреса, вечный поиск.
[www.trafaret.net-1701841625034.log](https://github.com/cdek-it/widget/files/13576789/www.trafaret.net-1701841625034.log)
![Screenshot 2023-12-06 084630](https://github.com/cdek-it/widget/assets/84074159/c634328d-97d5-4da9-9409-f4ca1c4bfed1)

**Comment by kaavain** [comment]
Короче плавает - в Мозилле сейчас тое карта даже не загрузилась. Указание
<script type='text/javascript' src='https://cdn.jsdelivr.net/gh/cdek-it/widget@3.7.0/dist/cdek-widget.umd.js' charset='utf-8'></script>
Приводит к работоспособности но с жуткими тромозами.

**Comment by kaavain** [comment]
PS: последнее на сегодня, случайно прогрузилась с 
<script type='text/javascript' src='https://cdn.jsdelivr.net/gh/cdek-it/widget@latest/dist/cdek-widget.umd.js' charset='utf-8'></script>
Но все так же залипает на курьерской при вводе адреса с домом.

**Comment by vermorag** [comment]
Новая версия становится доступна на CDN в течении недели. До этого времени будет подгружаться старая.

**Reply by kaavain** [reply]
Ранее мне говорили, что это не касается widget@latest, значит касается.
А то что карта не грузится ни в одном браузере - это связано с обновлением или нет?

**Reply by vermorag** [reply]
Firefox
![image](https://github.com/cdek-it/widget/assets/25276845/864a05d4-f9fb-4ecb-809a-ddcc942a4461)
Chrome
![image](https://github.com/cdek-it/widget/assets/25276845/674e300e-ab45-4079-9e32-0dd8fa3b2034)

**Reply by kaavain** [reply]
Уточню - "не грузится" != "вообще", а "практически" - то есть 1-2-3 минуты ожидания(1), в 30% случаев с ошибкой, в 70% в итоге да, но см (1).
